fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/OneRangePartitionEvaluator.java

```diff
@@ -102,31 +103,33 @@ public OneRangePartitionEvaluator(long partitionId, List<Slot> partitionSlots,
 
         PartitionRangeExpander expander = new PartitionRangeExpander();
         this.partitionSlotTypes = expander.computePartitionSlotTypes(lowers, uppers);
-        this.slotToType = IntStream.range(0, partitionSlots.size())
-                .mapToObj(index -> Pair.of(partitionSlots.get(index), partitionSlotTypes.get(index)))
-                .collect(ImmutableMap.toImmutableMap(Pair::key, Pair::value));
+        this.slotToType = Maps.newHashMapWithExpectedSize(partitionSlots.size() * 2);
```

Comment:
Minor: use Maps.newHashMapWithExpectedSize(partitionSlots.size()); may good enough, can update the code in next pr

Reply:
I will change to 16 in next pr, because the num of partitionSlots usually <= 3